### PR TITLE
increase ip formatting limits

### PR DESCRIPTION
**Pull request by tjungblu**
just had a must gather where we ran over the 1m IPs in the formatting, added a fix for that and some additional testing

### CODE REVIEW

File: pkg/obfuscator/mac_address.go [hunk 1]
```diff
@@ -58,7 +58,7 @@ func NewMacAddressObfuscator(replacementType schema.ObfuscateReplacementType) (O
 
 	reporter := NewSimpleTracker()
 	// creating a new generator object
-	generator := newGenerator(consistentMACTemplate, staticMacReplacement)
+	generator := newGenerator(consistentMACTemplate, staticMacReplacement, 9999999999)
```
Comment (sairameshv) [inline pkg/obfuscator/mac_address.go:61]: "maximumSupportedObfuscations" can be used in order to avoid hard-coding

Comment (tjungblu) [reply pkg/obfuscator/mac_address.go]: split it out into another constant to not accidentally mix things up

**Pull request merged**